org.eclipse.m2e.tests.common/src/org/eclipse/m2e/tests/common/HttpServer.java

```diff
@@ -657,17 +657,26 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
       for(String contextRoot : resourceDirs.keySet()) {
         String path = URIUtil.decodePath(trimContextRoot(uri, contextRoot));
         if(path != null) {
+
+          if(HttpMethod.CONNECT.asString().equals(request.getMethod())) {
```

Comment:
@laeubi can you tell if this is the right way to handle a CONNECT request (for example to `https://repo1.maven.org/maven2/`) in this local proxy server?

Reply:
Never mind, I just restored the few embedded artifacts for this case and reverted the change.